client/components/site-selector/index.jsx

```diff
@@ -104,19 +104,22 @@ class SiteSelector extends Component {
 			return;
 		}
 
-		if ( ! this.highlightedSiteRef ) {
-			selectorElement.scrollTop = 0;
-
+		if ( typeof document === 'undefined' ) {
 			return;
 		}
 
-		const highlightedSiteElement = ReactDom.findDOMNode( this.highlightedSiteRef );
+		// Note: Update CSS selectors if the class names change.
+		const highlightedSiteElem = document.body.querySelector(
+			'.site-selector .site.is-highlighted, .site-selector .all-sites.is-highlighted'
```

Comment:
Can you use `React.findDOMNode(this.refs.selector).querySelector()` to find the nodes instead of searching globally in `document`? It will search only for elements inside the current component.

That's more correct code. If there were two selectors rendered, searching `document.body` would scroll also selector #2 if you were navigating in selector #1. Very hypothetical bug, but I'd still prefer to scope the queries to the current component.

(and of course, rewrite the `this.refs.selector` ref to a function)

Reply:
Ah, that's a nice suggestion! Will do.

Reply:
Fixed in 841ec36.